### PR TITLE
Merge 185-linting-resolve-outstanding-linting-issues into develop

### PR DESCRIPTION
**Pull request by kylescudder**
## What

Resolved multiple linting issue throughout the project

## Why

To keep the code style consistent, this will make it easier in the long run as the project grows.

## How

Ran `pnpm lint` and resolved the issues that it reported.

## Checklist

- [x] I have tested these changes locally
- [x] I have ensured my code follows the project's coding style
- [x] I have updated the documentation accordingly
- [x] My changes do not introduce any new warnings or errors
- [x] All tests pass successfully
- [x] I have added/updated unit tests (if necessary)

### CODE REVIEW

**Review by f3rno64 — CHANGES_REQUESTED** [review]
We need to adjust the linter config to indent with spaces instead of tabs, and add a `format` command to the manifest to use [prettier](https://prettier.io/), along with pre-commit hooks to run the linter and tests prior to a commit being made. This is a separate task, one which I need to open an issue for, but I'm just noting it here.

For now, adjust the linter config to allow all-caps variable names for constants.

File: components/Features/Features.tsx [hunk 1]
```diff
@@ -3,19 +3,17 @@ import classNames from 'classnames'
...
 import {
-	FaChevronUp,
 	FaChevronLeft,
-	FaChevronDown,
 	FaChevronRight
 } from 'react-icons/fa'
 
 import { type FeaturesStage, FeaturesStageID } from './types'
 import {
-	FEATURES_STAGES,
```
[the remark this answers is not in the document]
Comment (f3rno64) [reply components/Features/Features.tsx:14]: Alright let's discuss on discord, I'll approve for now.

**Comment by kylescudder** [comment]
@f3rno64 if you are happy I am going to close this now as it has been superseded by #193  and #196.

**Comment by f3rno64** [comment]
Yep, go ahead.

**Pull request closed without merging**